src/cpu/aarch64/jit_uni_softmax.hpp

```diff
@@ -74,8 +74,10 @@ struct jit_uni_softmax_fwd_t : public primitive_t {
             const auto src_dt = src_md()->data_type;
             const auto dst_dt = dst_md()->data_type;
             bool ok = mayiuse(isa) && is_fwd() && !has_zero_dim_memory()
-                    && utils::one_of(src_dt, f32, s8, u8)
-                    && utils::one_of(dst_dt, f32, s8, u8)
+                    && utils::one_of(src_dt, f32, bf16, s8, u8)
+                    && utils::one_of(dst_dt, f32, bf16, s8, u8)
+                    && ((src_dt == bf16 || dst_dt == bf16) ? mayiuse_bf16()
+                                                           : true)
```

Comment:
Nit:
```suggestion
                    && IMPLICATION(utils::one_of(bf16, src_dt, dst_dt), mayiuse_bf16())
```

And same goes to backward.

Reply:
Hi @dzarukin, I have made the changes as per your suggestion in both the forward and backward SoftMax kernel initializations.